NOTES.md

```diff
@@ -1,3 +1,7 @@
+# Notes
+
+## Item Questions
+
 QUESTIONS ABOUT ITEMS
```

Comment:
Why rename this file? It's ancient (last touched 2011) and i have no idea if it's still relevant - could probably just delete it. Or leave it alone.

Reply:
it was for code organisation. but it's old, I guess we can remove it